doc/source/whatsnew/v0.18.1.txt

```diff
@@ -76,6 +76,25 @@ API changes
 
 
 
+- ``read_csv`` now raises an ``EmptyDataError``, a subclass of ``ValueError``, in response to empty columns or header (:issue:`12506`)
+
+.. ipython:: python
+
+   from pandas import read_csv
```

Comment:
don't need the first one, use ``pd.read_csv``

use ``from StringIO import StringIO``